scripts/create_xcframeworks.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Shebang to explicitly use bash and not break on macOS boxes with zsh as default shell.
+#
+# After that you usually describe what this script is for and how to use it. In particular,
+# what it expects the current directory to be. (That's more important if it's it "scripts".)
+
+set -eu # common flags to ensure that the shell does not ignore failures
+
+BUILD_PATH=build
+output_dir=$BUILD_PATH/xcf_output
```

Comment:
It would be nice to use
```bash
BUILD_PATH=${BUILD_PATH:-build}
```
so that the *default* value for this variable is `build` but it's possible to override it via environment, just like the Makefile allows it.

Reply:
got it, thanks